lms/templates/bulk_email/unsubscribe.html

```diff
@@ -0,0 +1,20 @@
+<%!
+from openedx.core.djangolib.markup import Text
+from django.utils.translation import ugettext as _
+%>
+
+<section class="container unsubscribe">
+
+  <section class="message">
+    <h1>${_("Unsubscribe Successful!")}</h1>
+    <hr class="horizontal-divider">
+
+    <p>
+      ${Text(
+        _("You have been unsubscibed from highlight emails for {course_key}. "
```

Comment:
This unsubscribe step will stop all scheduled emails, which currently includes highlights, nudges, and upgrade reminders.  Since this list might change, can you change this text to read: 
> You have been unsubscribed from scheduled emails for {course_key}.

Also, "unsubscibed" is a typo.